templates/estafette-entrypoint.sh

```diff
@@ -3,21 +3,21 @@ set -e
 
 {{- range .Commands }}
 {{- if .RunInBackground}}
-echo -e '\x1b[38;5;250m> {{.EscapedCommand}} &\x1b[0m'
+echo -e "\x1b[38;5;250m> {{.EscapedCommand}} &\x1b[0m"
 {{.Command}} &
 trap "kill $!; wait; exit" 1 2 15
-wait
+wait $!
```

Comment:
Wait without a process id doesn't return the exit code for that process, so adding the pid of the running command.